src/actions/SignupMain.js

```diff
@@ -2,7 +2,6 @@ export const GET_CODE_STATUS = "GET_CODE_STATUS";
 export const GET_CODE_STATUS_FAIL = "GET_CODE_STATUS_FAIL";
 export const APP_LOADED = "APP_LOADED";
 export const APP_LOADING = "APP_LOADING";
-export const APP_FETCHING = "APP_FETCHING";
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it

Reply:
Since the action is not used, the action type is neither